snackbars/src/main/java/com/decathlon/vitamin/compose/snackbars/VitaminSnackbarsColors.kt

```diff
@@ -0,0 +1,19 @@
+package com.decathlon.vitamin.compose.snackbars
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+data class VitaminSnackbarsColor(
+    val contentColor: Color,
+    val backgroundColor: Color
+)
+
+object VitaminSnackbarsColors {
+    val default: VitaminSnackbarsColor
```

Comment:
Rename the variable `primary` and use a function instead of a field and use a remember for Compose optimisation between recomposition.

```kotlin
@Composable
fun primary(
    contentColor: Color = VitaminTheme.colors.vtmnContentPrimaryReversed,
    backgroundColor: Color = VitaminTheme.colors.vtmnBackgroundPrimaryReversed
): SnackbarColors = remember(contentColor, backgroundColor) {
    SnackbarColors(contentColor = contentColor, backgroundColor = backgroundColor)
}
```